tests/test_events.py

```diff
@@ -29,42 +28,45 @@ async def _ignore_test_consoleLog_logEntryAddedEventEmitted(websocket):
             "expression": "console.log('some log message')",
             "target": {"context": context_id}}})
 
+    # The order of the two responses are not guarenteed
+    resp1 = await read_JSON_message(websocket)
+    resp2 = await read_JSON_message(websocket)
+    if "id" in resp1:
+        resp1, resp2 = resp2, resp1
+
     # Assert "log.entryAdded" event emitted.
-    resp = await read_JSON_message(websocket)
     recursiveCompare(
-        resp,
+        resp1,
         {
             "method": "log.entryAdded",
             "params": {
                 # BaseLogEntry
                 "level": "info",
-                "text": "some log message",
```

Comment:
What about step 5 from [`log.entryAdded`](https://w3c.github.io/webdriver-bidi/#event-log-entryAdded)?

Reply:
Implemented step 5 in commit 2 in file src\bidiMapper\domains\context\context.ts.

Reply:
Implemented formatter method to show preview of formatted texts.